admin/app/controllers/authenticated/certification-centers/get/invitations.js

```diff
@@ -49,6 +50,26 @@ export default class AuthenticatedCertificationCentersGetInvitationsController e
     this.isLoading = false;
   }
 
+  @action
+  async sendNewCertificationCenterInvitation(certificationCenterInvitation) {
+    this.isLoading = true;
```

Comment:
Ici il y a un `this.isLoading = true;` sans qu'il y ait le `this.isLoading = false;` correspondant.

Reply:
Il semble qu'il ne soit plus nécessaire de gérer les `isLoading` car ils sont gérés automatiquement dans certains cas par PixButton : 
> isLoading	
> Affiche un loader. Si @triggerAction retourne une promesse alors le loading est géré automatiquement.
>

cf. https://ui.pix.fr/?path=/docs/actions-button--docs

Donc il ne faut pas introduire de nouveaux `this.isLoading = true;` dans cette PR. Et idéalement les enlever tous de `admin/app/controllers/authenticated/certification-centers/get/invitations.js` dans un commit dédié.